core/shared/src/main/scala/zio/ZManaged.scala

```diff
@@ -1665,7 +1683,7 @@ object ZManaged extends ZManagedPlatformSpecific {
   def foreach_[R, E, A](as: Iterable[A])(f: A => ZManaged[R, E, Any]): ZManaged[R, E, Unit] =
     ZManaged(ZIO.foreach(as)(f(_).zio).map { result =>
       val (fins, _) = result.unzip
-      (e => ZIO.foreach(fins.reverse)(_.apply(e)), ())
+      (e => ZIO.foreach(fins.toList.reverse)(_.apply(e)), ())
```

Comment:
Seems like we can no longer work with `Iterable`. Could a low priority implicit build from compat work to restore that?

Reply:
I think this actually isn't an issue with our implementation at all. `reverse` isn't defined on `Iterable` so since we no longer automatically return a `List` we don't have a `Seq` that can be reverse.